pages/index.js

```diff
@@ -19,22 +20,33 @@ export default function Home({
     "Sonntag",
   ];
 
-  console.log(dailyQuizzesResultCollection);
-
   return (
     <Layout>
       <h2>Deine Woche</h2>
-      {weekDays.map((weekDay) => (
-        <WeekdayOptions
-          key={weekDay}
-          setSelectedFoodCategories={setSelectedFoodCategories}
-          dailyQuizzesResultCollection={dailyQuizzesResultCollection}
-          setDailyQuizzesResultCollection={setDailyQuizzesResultCollection}
-          weekDay={weekDay}
-          setCurrentWeekDay={setCurrentWeekDay}
-          currentWeekDay={currentWeekDay}
-        />
-      ))}
+      <List>
```

Comment:
To ensure a11y in Safari as well because you've set `list-style: none`;
```suggestion
      <List role="list">
```

Reply:
integrated that now, thank you!!